src/modules/auth/api/index.api.ts

```diff
@@ -0,0 +1,12 @@
+import axios from '@shared/utils/axios.util';
+import { TrainerSignup, Login } from '../types';
+
+const BASE_URL = 'auth';
```

Comment:
What do you think about naming this file `auth.api.ts`? 
I think it would be easier to find with a search...